server/routes/barcode/image/GenerateBarcodeImageController.ts

```diff
@@ -18,11 +19,21 @@ export default class GenerateBarcodeImageController {
         req.session.createBarcodeAuthToken
       )
 
-      const recipient = req.session.recipients[0]
-      const barcodeImageDataUrl = await this.createBarcodeService.generateAddressAndBarcodeDataUrlImage(recipient)
-      const barcodeImageName = this.barcodeFilename(recipient)
+      const barcodeImages = await Promise.all(
+        req.session.recipients.map(async recipient => {
+          try {
+            return {
+              barcodeImageUrl: await this.createBarcodeService.generateAddressAndBarcodeDataUrlImage(recipient),
+              barcodeImageName: this.barcodeFilename(recipient),
+            }
+          } catch (error) {
+            logger.error(`Could not generate a barcode for ${recipient.prisonNumber}, ${error}`)
+            return undefined
```

Comment:
I've not addressed error handling in this PR, though I fully intend to as part of the overall ticket.
I'm going to get a conversation going with Liora about error handling. Not so much the error messaging, but more what to do if 1 fails - do we fail just that one but carry on, or do we fail them all?